services/data-info/src/data_info/services/exists.clj

```diff
@@ -18,7 +18,7 @@
   [{user :user} {paths :paths}]
   (with-jargon (cfg/jargon-cfg) [cm]
     (duv/user-exists cm user)
-    {:paths (into {} (map (juxt identity (partial path-exists-for-user? cm user)) (set paths)))}))
+    {:paths (into {} (map (juxt keyword (partial path-exists-for-user? cm user)) (set paths)))}))
```

Comment:
This is what we'd actually specified for these, so I made them keywords rather than strings (with identity). Seems to work fine even though the created keywords can't be literally created with the `:` syntax due to slashes.